### ISSUE TITLE
Fever greater than or equal to 38 Celsius

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7112]
**Preferred term label:**
Fever greater than or equal to 38 Celsius

**Synonyms**
Fever >= 38°C
Fever greater than or equal to 100.4 Fahrenheit
Fever >= 100.4°F
Pyrexia >= 38°C
Pyrexia greater than or equal to 100.4 Fahrenheit
Pyrexia >= 100.4°F

**Definition (free text, please give PubMed ID)**
Fever that is or exceeds 38 degrees centigrade. [PMID: 27581757]

**Parent term (use hpo.jax.org/app)**
HP:0001945 - Fever

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
This is being requested for use with COVID-19 clinical contextual data.

**Your nano-attribution (ORCID)**
0000-0002-9578-0788

---
**Note to issue assignees:** Is not synonymous with [SNOWMEDCT_US:426000000](https://phinvads.cdc.gov/vads/ViewCodeSystemConcept.action?oid=2.16.840.1.113883.6.96&code=426000000) as this code is not inclusive ,only greater than. Decided to mimic it's naming convention for the term label, but change it to Celsius since it appears in more HP terms than Fahrenheit (at a cursory glance).

**Comment by pnrobinson** [comment]
This is equivalent to the existing term [Fever HP:0001945](https://hpo.jax.org/app/browse/term/HP:0001945).
The HPO does not attempt to model numerical data, and although there is no universally accepted temperature threshold for fever, most sources use 38 degrees. 
If your analysis needs to have the numbers, it is probably best to annotate with the HPO term Fever and store the actual measured temperatures in an additional field.
